tests/test_main.py

```diff
@@ -177,18 +142,64 @@ def test_trim_pair_no_hmmer():
                                                     taxa=taxa,
                                                     region=region))
 
-
-def test_trim_pair_no_bb():
-    sequences, single_end = _itsxpress._fastq_id_maker(per_sample_sequences=TEST_DATA,
-                                                       artifact_type=ARTIFACT_TYPE_P)
-    for sequence in sequences:
-        raises(ValueError, lambda: _itsxpress._set_fastqs_and_check(per_sample_sequences=TEST_DATA,
-                                                                    artifact_type=ARTIFACT_TYPE_P,
-                                                                    sequence=sequence,
-                                                                    single_end=single_end,
-                                                                    threads=1))
-        raises(ValueError, lambda: _itsxpress._set_fastqs_and_check(per_sample_sequences=TEST_DATA,
-                                                                    artifact_type=ARTIFACT_TYPE_S,
-                                                                    sequence=sequence,
-                                                                    single_end=single_end,
-                                                                    threads=1))
+class TrimTests(unittest.TestCase):
```

Comment:
These tests invoke QIIME 2 directly --- this not only tests the method functionality, but also ensures things like plugin registration is sound.